dev/NavigationView/TopNavigationViewDataProvider.cpp

```diff
@@ -131,6 +131,17 @@ std::vector<int> TopNavigationViewDataProvider::ConvertPrimaryIndexToIndex(std::
     return indexes;
 }
 
+int TopNavigationViewDataProvider::ConvertOriginalIndexToIndex(int originalIndex)
+{
+    auto vector = GetVector(PrimaryList);
+    if (!IsItemInPrimaryList(originalIndex))
+    {
+        vector = GetVector(OverflowList);
+    }
```

Comment:
suggestion:

```
auto const vector = IsItemInPrimaryList(originalIndex) ? GetVector(PrimaryList) : GetVector(OverflowList)
```

This allows you to define your variable as const and only calls GetVector once.